modules/nf-core/pairtools/sample/main.nf

```diff
@@ -0,0 +1,38 @@
+process PAIRTOOLS_SAMPLE {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::pairtools=0.3.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pairtools:0.3.0--py37hb9c2fc3_5' :
+        'quay.io/biocontainers/pairtools:0.3.0--py37hb9c2fc3_5' }"
+
+    input:
+    tuple val(meta), path(input), val(fraction)
+
+    output:
+    tuple val(meta), path("*.pairsam.gz")  , emit: pairsam
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}_subsample"
```

Comment:
It's preffered to keep the default of the `prefix` as `meta.id`. The guidelines state that for modules that can have the same input and output, you should add a check with an error message: see [here](https://nf-co.re/docs/contributing/modules#naming-conventions) paragraph 6